README.md

```diff
@@ -2,7 +2,7 @@
 
 # Rails-React App
 
->  The application is a test of combining rails API and a react application
+>  The application is a test of combining rails API and a react to make a full stack app
```

Comment:
- [ ] Unfortunately, I cannot see the api populated with the message. Kindly add a message if needed as an example in seed ![image](https://user-images.githubusercontent.com/62705947/189680954-0f5253cc-3e86-4bd7-bf53-41c967fd5ae4.png)